pages/windows/logoff.md

```diff
@@ -0,0 +1,19 @@
+# logoff
+
+> Terminate a login session.
+
+- Terminate the current session:
+
+`logoff`
+
+- Terminate a session by its name or id:
+
+`logoff {{session_name|session_id}}`
+
+- Terminate a session on a specific server connected through RDP:
+
+`logoff /server:{{servername}}`
```

Comment:
How does it know which session to terminate on the remote server?

Reply:
It needs to be specified, I'll add that in.